src/wing/CCPACSWing.h

```diff
@@ -64,125 +77,250 @@ friend class CTiglWingBuilder;
     TIGL_EXPORT void SetSymmetryAxis(const TiglSymmetryAxis& axis) override;
     TIGL_EXPORT void SetParentUID(const boost::optional<std::string>& value) override;
 
-    // Returns whether this wing is a rotor blade
+    /**
+     * @brief Returns whether this wing is a rotor blade
+     * @return bool
+     */
     TIGL_EXPORT bool IsRotorBlade() const;
 
-    // Returns the parent configuration
+    /**
+     *  @brief Returns the parent configuration
+     *  @return CCPACSConfiguration&
+     */
     TIGL_EXPORT CCPACSConfiguration & GetConfiguration() const;
 
-    // Get section count
+    /**
+     * @brief Get section count
+     * @return int
+     */
     TIGL_EXPORT int GetSectionCount() const;
 
-    // Returns the section for a given index
+    /**
+     * @brief Returns the section for a given index
+     * @return CCPACSWingSection&
+     */
     TIGL_EXPORT CCPACSWingSection& GetSection(int index);
     TIGL_EXPORT const CCPACSWingSection& GetSection(int index) const;
 
-    // Getter of the number of segments of the wing
+
+    /**
+     *  @brief Returns the number of segments of the wing
+     *  @param index
+     *  @return int
+     */
     TIGL_EXPORT int GetSegmentCount() const;
 
-    // Returns the segment for a given index or uid
+    /**
+     * @brief Returns the segment for a given index or uid
```

Comment:
I believe this docstring only applies to the first of the four functions. Each overload must get its own docstring, does it not?

Reply:
```suggestion
     * @brief Returns the segment for a given index
```